worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -111,8 +111,12 @@ class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, Role.WORKER)
     val applicationIds = appActiveConnections.get(client.getChannel.id().asLongText())
     val applicationId = Utils.splitShuffleKey(shuffleKey)._1
     if (applicationIds != null && !applicationIds.contains(applicationId)) {
-      addCounter(ACTIVE_CONNECTION_COUNT, Map(applicationLabel -> applicationId))
-      incCounter(ACTIVE_CONNECTION_COUNT, 1, Map(applicationLabel -> applicationId))
+      var labels = Map.empty[String, String]
+      if (conf.metricsPushApplicationId) {
+        labels += (applicationLabel -> applicationId)
+        addCounter(ACTIVE_CONNECTION_COUNT, labels)
```

Comment:
This metric undoubtedly has high cardinality, it is better that we can disable this metrics with config.

At the same time, I may want to push top application consumption metrics.

How do you think about? @pan3793  @SteNicholas

Reply:
For top app consumption metrics, just rely on the `celeborn.metrics.worker.app.topResourceConsumption.count`? Set 0 to disable it?

Reply:
@turboFei Adding an API for checking top application consumption on a worker can be helpful for debugging the usage, i filed a ticket for the same earlier – https://issues.apache.org/jira/browse/CELEBORN-1975

In our case, for debugging we check userIdentifier consumption and figure out applicationId using spark listener audit logs.

Reply:
The TOP  application consumption was remove by https://github.com/apache/celeborn/pull/2949